app/views/spree/orders/order_cycle_expired.html.haml

```diff
@@ -2,7 +2,7 @@
   = t :orders_oc_expired_headline
 
 %p
-  - if active_order_cycle_for_distributor? current_distributor
+  - if active_order_cycle_for_distributor? @distributor
```

Comment:
Why the change from `current_distributor` to `@distributor` ?

Reply:
The helper method was ignoring the argument (which was `current_distributor`) and used `@distributor` instead. Now we are passing that on and use the argument.

When looking at an order, we need to display information about the distributor of the order and not of the `current_distributor` which is the shop you selected last. It may be different.

Reply:
I see ! thanks for the clarification.